app/assets/stylesheets/darkswarm/modals.css.scss

```diff
@@ -18,23 +16,26 @@ dialog
   // Reveal.js break point:
   // @media only screen and (max-width: 40.063em)
 
-  // Small - when modal IS full screen
+  // Small - smaller outside area
   @media only screen and (max-width: 640px) {
-    left: 0;
-    max-height: 100%;
-    position: absolute !important;
-    top: 0;
+    left: 4%;
+    max-height: 92%;
+    max-width: 92%;
+    padding: 15px 0 0;
+    top: 4%;
   }
 
-  // Medium and up - when modal IS NOT full screen
+  // Medium and up - larger outside area
   @media only screen and (min-width: 641px) {
+    border-bottom: 30px solid white;
```

Comment:
don't we have a var for `white`? if we ever decide we go with a slightly off-white (very common) we'll have to go replace all uses of `white`.

Reply:
I see we already do that with `$black` :point_down:

Reply:
We do have $white already.

Reply:
We do have $white already.

Reply:
:+1: